src/main/java/com/grimacegram/grimacegram/services/UserService.java

```diff
@@ -0,0 +1,35 @@
+package com.grimacegram.grimacegram.services;
+
+import com.grimacegram.grimacegram.model.User;
+import com.grimacegram.grimacegram.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Service;
+
+@Service
+public class UserService {
+
+    UserRepository userRepository;
+
+    PasswordEncoder passwordEncoder;
+
+    @Autowired
+    public UserService(UserRepository userRepository, PasswordEncoder passwordEncoder) {
+        this.userRepository = userRepository;
+        this.passwordEncoder = passwordEncoder;
+    }
+
+    public User save(User user){
+        user.setPassword(passwordEncoder.encode(user.getPassword()));
+        return userRepository.save(user);
+    }
+
+    public Page<?> getUsers() {
+        Pageable pageable = PageRequest.of(0, 10);
```

Comment:
It would a nice exercise to make size and offset of PageRequest changeable from the front-end.